### ISSUE TITLE
confirmation bypass option for gameserver console command

### ISSUE THREAD

**Issue by kriegtiger** [GameServerManagers/LinuxGSM#2936]
## User Story

When setting up my LGSM arkserver instance, I used the console multiple times, and it was mildly bothersome to run the './arkserver console' command, then have to wait and input confirmation, before I could actually see the console.

## Basic info

* **Distro:** Debian 9
* **Game:** ARK
* **Command:** ./arkserver console
* **LinuxGSM version:** v20.3.3

## Further Information

It would be nice if it were possible to issue a '-y' or something along with the console command (ie: ./arkserver -y console ) in order to skip past that additional prompt, or maybe a shell variable that is effectively a 'usage understood' EULA style nod that says 'I know how to exit/use the console, skip ahead please'.

## Further Reading

Numerous Linux commands/utilities have '-y' functionality to automatically say 'yes' to whatever prompt may be presented

**Comment by dgibbs64** [comment]
So the reason a Y is added is to allow users to ability to understand how to correctly exit the console. 

I think there are a couple of options here. Either add a count down timer of say 5 seconds. Or try to speed the messaging to get to the Y point. Im not to sure on implementing a -y option currently

**Comment by kriegtiger** [comment]
> So the reason a Y is added is to allow users to ability to understand how to correctly exit the console.
> 
> I think there are a couple of options here. Either add a count down timer of say 5 seconds. Or try to speed the messaging to get to the Y point. Im not to sure on implementing a -y option currently

Yep, I get that. It's a perfectly reasonable reminder to have for a while to remind people how it works, it's just an irritant once you're familiar with the process - hence the request for having a way to bypass it later if the user wishes.

What about having something like '<CTRL-B>, D to exit' in the green bar at the bottom that is (at least for ARK?) mostly empty? Or maybe add a second green bar at the top with usage reminders. Information is still conveyed and will *always* be present in the user's vision in this case instead of just a blip at the beginning.

**Comment by kriegtiger** [comment]
> 
> 
> Just saw this issue was still open - what about something like a `skipconsolewarning` variable, defaulting to not skip it? I'd be willing to submit a pull request for it. It's something that'd only be known by those who already know about it, anyway - ie. you'd have to dig into the config to see it, indicating you already know how to use LinuxGSM, and we can have how to exit be part of the config value description.

That sounds good to me. Would it be possible to have that bit in the green bar at the bottom as a reminder of how to properly exit the console? Not sure if that area is reserved for other functions/data depending on the game being served.

**Comment by soccermitchy** [comment]
In theory it'd be possible, but I'm not sure if that'd be something the LinuxGSM team would be willing to do on the official versions of LGSM. The bottom bar is from `tmux`, and a lot of people use custom tmux configurations which may be setting their own custom text, and we'd be trampling on over the configuration if we set it. That said, it's literally one command to do it:
`tmux set -t $sessionname status-right " [<Ctrl-b> d to exit] "#{=21:pane_title}" %H:%M %d-%b-%y"`
Or from within the session itself:
<Ctrl-b> `:set status-right " [<Ctrl-b> d to exit] "#{=21:pane_title}" %H:%M %d-%b-%y"`

**Comment by soccermitchy** [comment]
Re: tmux configuration, you can put this in your `~/.tmux.conf` for a similar thing:
```tmux
set -g status-right " [<Ctrl-b> d to exit] "#{=21:pane_title}" %H:%M %d-%b-%y"
```

**Comment by rtxa** [comment]
@dgibbs64 Sorry for tagging you. Any updates on this issue? I find a good solution to add some option like `skip-console-warning` to avoid the annoyance to keep pressing Y everytime you want to access the console, start the server, etc. when you are already familiarized with LinuxGSM.
